docs/source/loggers/tensorboard_logger.md

```diff
@@ -0,0 +1,198 @@
+# Logging Your Results with Tensorboard
+
+## Installing Tensorboard
+Before we get started make sure you have installed Tensorboard!
+Make sure to run:
+```bash
+pip install 'mosaicml[tensorboard]'
+```
+
+## Logging to Tensorboard Pythonically
+To log your run's results to tensorboard, first you will need to create a `TensorboardLogger`
+object, like so:
+
+```python
+from composer.loggers import TensorboardLogger
+
+tb_logger = TensorboardLogger(log_dir="./my_tensorboard_logs")
+```
+
+`log_dir` is where you want the Tensorboard logs to be saved locally (on the system in which you run composer).
+
+ If you are viewing or accessing you logs locally, choose this path wisely and remember it! Also make sure to use this same value for `log_dir` for any future runs (so all your runs can be visualized together!)
+ 
+ If you will be using S3 to save your logs then the exact path you choose is not as important.
```

Comment:
Should link to the `ObjectStoreLogger` docs and mention that the path passed to there is what is used

Reply:
done